index.css

```diff
@@ -41,12 +41,13 @@ p {
     margin: 0;
 }
 
-#wrapper {
+#sub-menu-wrapper {
     border: 2px solid red;
-    height: 463px;
+    height: 94%;
```

Comment:
As mentioned before, setting height is never great - you could use padding on parrent

Reply:
Solved